force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -28,10 +28,22 @@ public with sharing class FlowControlLoopsCollections {
      * @param y The second number.
      * @return "Hello World!" if x greater than y, otherwise return null.
      */
-    public static String helloWorld(Integer x, Integer y) {
-        return null; // Replace null with the variable you used to store the result
-    }
-
+        public static String helloWorld(Integer x, Integer y) {
+            if (x == null || y == null) {
```

Comment:
Good job checking for both in the same line, and for putting the null check first.